vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/Slide.java

```diff
@@ -27,23 +28,25 @@
 
     @Id
     @GeneratedValue(generator = "slide_id_generator")
-    @GenericGenerator(name = "slide_id_generator", 
-        parameters = @Parameter(name = "prefix", value = "SLI"),
-        strategy = "edu.asu.diging.vspace.core.data.IdGenerator")
+    @GenericGenerator(name = "slide_id_generator", parameters = @Parameter(name = "prefix", value = "SLI"), strategy = "edu.asu.diging.vspace.core.data.IdGenerator")
     private String id;
 
     @ManyToOne(targetEntity = Module.class)
     private IModule module;
 
-    //-------- @JsonIgnore used as this entity will be returned in a controller
-    @JsonIgnore 
+    // -------- @JsonIgnore used as this entity will be returned in a controller
+    @JsonIgnore
     @OneToMany(targetEntity = ContentBlock.class, mappedBy = "slide", cascade = CascadeType.ALL)
     private List<IContentBlock> contents;
-    
+
     @JsonIgnore
     @ManyToMany(mappedBy = "slides", targetEntity = Sequence.class)
     private List<ISequence> sequence;
 
+    @JsonIgnore
+    @Transient
+    private ImageBlock firstImageBlock;
```

Comment:
this doesn't seem to be used. remove.

Reply:
if I am removing this then I am getting Jackson data bind exception.